app/models.py

```diff
@@ -43,11 +43,6 @@
 PUSH_TYPE = 'push'
 SMS_TYPE = 'sms'
 
-VA_NOTIFY_TO_VA_PROFILE_NOTIFICATION_TYPES = {
-    EMAIL_TYPE: 'Email',
-    SMS_TYPE: 'Text',
-}
-
```

Comment:
See [this convo](https://github.com/department-of-veterans-affairs/notification-api/pull/1880/files#r1700646789). in order to use this dict and prevent circular import, we had to do an import within the body of a function.  This has now been moved over to va_profile_client.py